src/components/BotBuilder/BotBuilderPages/DesignViews/UIView.js

```diff
@@ -483,9 +502,47 @@ class UIView extends Component {
             <Row>
               <Col xs={12} md={6} lg={6}>
                 {component.id === 7 ? (
-                  <h2>{component.name}</h2>
+                  <div style={{ display: 'flex', flexDirection: 'column' }}>
+                    <div
+                      style={{
+                        fontSize: '18px',
+                        paddingTop: '12px',
+                        fontWeight: '400',
+                      }}
+                    >
+                      {component.name}
+                    </div>
+                    <div
+                      style={{
+                        fontSize: '13px',
+                        paddingTop: '12px',
+                        fontWeight: '100',
+                      }}
+                    >
+                      {component.helperText}
+                    </div>
+                  </div>
                 ) : (
-                  <h2>Color of {component.name}</h2>
+                  <div style={{ display: 'flex', flexDirection: 'column' }}>
+                    <div
+                      style={{
+                        fontSize: '18px',
+                        paddingTop: '12px',
+                        fontWeight: '400',
+                      }}
+                    >
+                      Color of {component.name}
+                    </div>
+                    <div
+                      style={{
+                        fontSize: '13px',
+                        paddingTop: '12px',
+                        fontWeight: '100',
```

Comment:
Can you please try changing the font weight to `300` or increase the font size? Its seems unreadable now.